recipe-client-addon/content/about-studies/.eslintrc.js

```diff
@@ -0,0 +1,15 @@
+module.exports = {
+  globals: {
+    classnames: false,
+    FxButton: false,
+    InfoBox: false,
+    PropTypes: false,
+    r: false,
+    React: false,
+    ReactDOM: false,
+    remoteValues: false,
+    sendPageEvent: false,
+    ShieldStudies: false,
+    stringToDate: false,
```

Comment:
This, and the bunch of `<script>` tags make me nervous. (BuddyUp flashbacks, fwiw). Is it possible to use Cu.import or some other module system in the JS files?